decidim-elections/app/commands/decidim/elections/admin/setup_election.rb

```diff
@@ -66,6 +66,7 @@ def election_data
           {
             iat: Time.now.to_i,
             election_id: election_id,
+            message_id: "#{election_id}.create_election+a.#{bulletin_board.authority_slug}",
             type: "create_election",
```

Comment:
we can remove the `election_id` and `create_election`, as they are included in the `message_id`

Reply:
Yeah, I have that in mind. I wanted to do that with the refactoring of moving the whole `election_data` to the gem.